tests/snuba/api/endpoints/test_organization_events_stats_mep.py

```diff
@@ -1080,7 +1080,8 @@ def test_top_events_with_transaction_on_demand(self):
         for group_count in groups:
             group, agg, row1, row2 = group_count
             row_data = response.data[group][agg]["data"][:2]
-            assert [attrs for time, attrs in row_data] == [[{"count": row1}], [{"count": row2}]]
+            assert [attrs for _, attrs in row_data] == [[{"count": row1}], [{"count": row2}]]
+            assert response.data[group][agg]["meta"]["isMetricsExtractedData"]
```

Comment:
@k-fish Should I place this at the root of the API response rather than on each element under `data`?